sdk/cognitivelanguage/Azure.AI.Language.Text/samples/Sample7_AnalyzeTextOperation_HealthcareOperationAction.md

```diff
@@ -0,0 +1,153 @@
+# Analyze healthcare entities
```

Comment:
How is this sample different from the analyzeTextOperationAsync for healthcare operation?

Reply:
It is calling the non async function..this sample is mirrored from the old sdk

Reply:
Do we really need this ? Since healthcare doesn't have anything other than Async

Reply:
it is C# notion of async not LRO async